src/lib/core/option/option.ts

```diff
@@ -99,17 +92,22 @@ export class MdOption extends _MdOptionMixinBase implements CanDisableRipple {
   get disabled() { return (this.group && this.group.disabled) || this._disabled; }
   set disabled(value: any) { this._disabled = coerceBooleanProperty(value); }
 
+  /** Whether ripples for the option are disabled. */
+  @Input()
```

Comment:
The ability to disable the ripples from a binding has been already added in https://github.com/angular/material2/commit/addf1ce8b124fe7a242bf927e59ea69d00513f85. I don't really see anything that would go against just providing this flexibility.